org.kde.isoimagewriter.json

```diff
@@ -0,0 +1,35 @@
+{
+   "id": "org.kde.isoimagewriter",
+   "runtime": "org.kde.Platform",
+   "runtime-version": "5.15-22.08",
+   "sdk": "org.kde.Sdk",
+   "command": "isoimagewriter",
+   "finish-args": [
+      "--device=dri",
+      "--filesystem=host",
+      "--share=ipc",
+      "--socket=fallback-x11",
+      "--socket=wayland",
+      "--system-talk-name=org.freedesktop.UDisks2",
+      "--talk-name=org.freedesktop.Flatpak"
+   ],
+   "modules": [
+      {
+         "buildsystem": "cmake-ninja",
+         "name": "isoimagewriter",
+         "sources": [
+            {
+               "type": "archive",
+               "url": "https://download.kde.org/stable/isoimagewriter/1.0.0/isoimagewriter-1.0.0.tar.xz",
+               "sha256": "a69022303ec1beb6b7b4f0d68e59e4199d3c9868767cb9eb23c6dd1999e06a5d",
+               "x-checker-data": {
+                     "type": "anitya",
+                     "project-id": 8763,
```

Comment:
This is wrong https://release-monitoring.org/project/8763 is not isoimagewriter 

https://release-monitoring.org/project/8763

Reply:
how do I find out the project-id?

Reply:
Use the search field/button on the site.

But I can see it not existing so you have three options: 
 - Not have a x-checker-data
 - Add kde iso image writter to release-monitoring.org
 - Use a different x-checker-data backend